kie-api/src/test/java/org/kie/api/internal/utils/AnotherMockAssemblersImpl.java

```diff
@@ -14,5 +14,6 @@
 
 package org.kie.api.internal.utils;
 
+@ServicePriority( 4 )
```

Comment:
@mariofusco 
I strongly think we should not do this kind of stuff, as I tried to explain.